release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1464,6 +1476,18 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
 [id="ocp-4-11-routing-bug-fixes"]
 ==== Routing
 
+* Previously, the Ingress Operator did not validate whether a Kubernetes service object in the OpenShift Ingress namespace was created or owned by the Ingress Controller it was trying to reconcile with. Therefore, the Ingress Operator would modify or remove Kubernetes services that had the same name and namespace, regardless of ownership, causing unexpected behavior. With this update, the Ingress Operator can now check the ownership of existing Kubernetes services before attempting to modify or remove services. If ownership does not match, the Ingress Operator shows an error and does not take any action. As a result, the Ingress Operator cannot cannot modify or delete a custom Kubernetes service with the same name as the OpenShift Ingress namespace that it wants to modify or remove. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054200[*BZ#2054200*])
+
+* Previously, {product-title} 4.8 added an API for customizing platform routes. This API includes status and spec fields in the cluster ingress configuration for reporting the current host names of customizable routes as well as the user's desired host names for these routes, respectively. The API also defined constraints for these values. These constraints were restrictive and excluded some valid potential host name. Consequently, the restrictive validation for the API prevented users from specifying custom host names that should have been permitted, and prevented users from being able to install clusters with domains that should have been permitted. With this update, the constraints on host names were relaxed to allow all host names that are valid for routes and the {product-title} allows users to use cluster domains with TLDs that contain decimal digits. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039256[*BZ#2039256*])
+
+* * Previously, the Ingress Operator did not check whether Ingress Controllers with spec.domains matched the spec.baseDomain. This caused the Operator to create DNS records and set `DNSManaged` conditions to false. With this fix, the Ingress Operator now checks whether the spec.domain matches with the cluster's spec.baseDomain. As a result, the Ingress Operator does not create DNS records and sets `DNSManaged` conditions to false for custom Ingress Controllers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2041616#[*BZ#2041616*])
```

Comment:
s/with spec.domains/configured with the cluster `spec.domain` parameter (something like that, note  no _s_ in domain)
s/spec.baseDomain/`spec.baseDomain` parameter
s/false/`false` ??
s/whether the spec.domain/whether the `spec.domain` parameter
s/cluster's spec.baseDomain/cluster spec.baseDomain
Should _for custom Ingress Controllers_ come at the beginning of the sentence for clarity?
_As a result, for custom Ingress Controllers,  the Ingress Operator...._